ui/app/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -42,6 +48,24 @@ export default class PermissionConnect extends Component {
     originName: this.props.originName,
   }
 
+  componentDidUpdate (prevProps) {
+    const { permissionsRequest, domains } = this.props
+    const { originName } = this.state
+
+    if (permissionsRequest === null && prevProps.permissionsRequest && prevProps.permissionsRequest.permissions) {
```

Comment:
Can `permissionsRequest` ever be `null`? It has a default prop of `{}`

Edit: Oh I see, the selector returns null, and the `defaultProp` is only used if it's given `undefined`. To make this easier to understand, I'd suggest that we set the default prop to `undefined`.

Reply:
1b68fb882